features/google/rollup.config.mjs

```diff
@@ -78,7 +78,7 @@ export default (commandLineArgs) => {
             // overwite the default warning function
             if (
                 warning.code === "CIRCULAR_DEPENDENCY" &&
-                warning.cycle[0].match(/(d3-|importer-context)/)
+                warning.cycle[0].match(/(d3-|google-context)/)
```

Comment:
You sure about this one? You haven't changed anything in here. Or it was an error from before?

Reply:
Copy paste error from before I reckon